heuristic/classes/Item.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+
+
+class Item:
+    volume: float
+    customer: int
+
+    @classmethod
+    def create(cls, volume, customer) -> Item:
```

Comment:
This can be ```__init__``` - it's a regular constructor (I use these classmethods sometimes for more specific ways to construct an object, but those should be used.. sparingly).

Reply:
Also types